### PR TITLE
Fix: Navigation panel is attached to the top in Editor Top Bar [ED-10481]

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: packages/packages/recently-edited/src/components/top-bar/indicator.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function Indicator( { title, status }: Props ) {
 	return (
 		<Tooltip title={ title }>
 			<Stack direction="row" alignItems="center" spacing={ 2 }>
-				<Typography variant="body2" sx={ { maxWidth: '120px' } } noWrap>
+				<Typography variant="body2" sx={ { alignSelf: 'stretch', maxWidth: '120px' } } noWrap>
```
Comment (rotemee) [inline packages/packages/recently-edited/src/components/top-bar/indicator.tsx:13]:
```suggestion
				<Typography variant="body2" sx={ { maxWidth: '120px' } } noWrap>
```

File: packages/packages/recently-edited/src/components/top-bar/recently-edited.tsx [hunk 2]
```diff
@@ -32,9 +32,10 @@ export default function RecentlyEdited() {
 	}
 
 	return (
-		<Stack direction="row" spacing={ 1 } alignItems="center" sx={ { paddingInlineStart: 1, cursor: 'default' } }>
+		<Stack direction="row" spacing={ 1 } alignItems="center" sx={ { alignSelf: 'stretch', paddingInlineStart: 1, cursor: 'default' } }>
```
Comment (rotemee) [inline packages/packages/recently-edited/src/components/top-bar/recently-edited.tsx:35]: Needed?

**Review by rotemee — CHANGES_REQUESTED** [review]

File: packages/packages/recently-edited/src/components/top-bar/indicator.tsx (hunk 1 above)
Comment (ManorHazazE) [reply packages/packages/recently-edited/src/components/top-bar/indicator.tsx]: Done

File: packages/packages/recently-edited/src/components/top-bar/recently-edited.tsx (hunk 2 above)
Comment (ManorHazazE) [reply packages/packages/recently-edited/src/components/top-bar/recently-edited.tsx]: Removed

**Review by rotemee — DISMISSED** [review]

**Pull request closed without merging**